diff --git a/menu/drivers/ozone.c b/menu/drivers/ozone.c
index 2c26c12b6c..9a9c6335c9 100644
--- a/menu/drivers/ozone.c
+++ b/menu/drivers/ozone.c
@@ -5203,20 +5203,20 @@ static void ozone_refresh_system_tabs_list(ozone_handle_t * ozone)
       ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_VIDEO;
 #endif

-   if (      settings->uints.menu_content_show_add_entry == MENU_ADD_CONTENT_ENTRY_DISPLAY_PLAYLISTS_TAB
-         && !settings->bools.kiosk_mode_enable)
-      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_ADD;
-
-#if defined(HAVE_DYNAMIC)
-   if (settings->uints.menu_content_show_contentless_cores !=
-         MENU_CONTENTLESS_CORES_DISPLAY_NONE)
-      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_CONTENTLESS_CORES;
-#endif
-
-#if defined(HAVE_LIBRETRODB)
-   if (settings->bools.menu_content_show_explore)
-      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_EXPLORE;
-#endif
+//   if (      settings->uints.menu_content_show_add_entry == MENU_ADD_CONTENT_ENTRY_DISPLAY_PLAYLISTS_TAB
+//         && !settings->bools.kiosk_mode_enable)
+//      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_ADD;
+
+//#if defined(HAVE_DYNAMIC)
+//   if (settings->uints.menu_content_show_contentless_cores !=
+//         MENU_CONTENTLESS_CORES_DISPLAY_NONE)
+//      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_CONTENTLESS_CORES;
+//#endif
+
+//#if defined(HAVE_LIBRETRODB)
+//   if (settings->bools.menu_content_show_explore)
+//      ozone->tabs[++ozone->system_tab_end] = OZONE_SYSTEM_TAB_EXPLORE;
+//#endif
 }

 static int ozone_get_entries_padding_old_list(ozone_handle_t* ozone)
diff --git a/menu/menu_setting.c b/menu/menu_setting.c
index 6371398a27..42e23bae2b 100644
--- a/menu/menu_setting.c
+++ b/menu/menu_setting.c
@@ -9199,7 +9199,7 @@ static void update_streaming_url_write_handler(rarch_setting_t *setting)
    recording_driver_update_streaming_url();
 }

-#ifdef HAVE_LAKKA
+#ifndef HAVE_LAKKA
 static void systemd_service_toggle(const char *path, char *unit, bool enable)
 {
    pid_t pid    = fork();
@@ -9261,6 +9261,7 @@ static void bluetooth_ertm_disable_toggle_change_handler(rarch_setting_t *settin
 }
 #endif

+#ifndef HAVE_LAKKA
 static void ssh_enable_toggle_change_handler(rarch_setting_t *setting)
 {
    systemd_service_toggle(LAKKA_SSH_PATH, (char*)"sshd.service",
@@ -9308,6 +9309,7 @@ static void timezone_change_handler(rarch_setting_t *setting)
    }
 }
 #endif
+#endif

 static void appicon_change_handler(rarch_setting_t *setting)
 {
@@ -20587,7 +20589,7 @@ static bool setting_append_list(
                SD_FLAG_CMD_APPLY_AUTO);
 #endif

-#ifdef HAVE_LAKKA
+#ifndef HAVE_LAKKA
          CONFIG_ACTION(
                list, list_info,
                MENU_ENUM_LABEL_CPU_PERFPOWER,
@@ -23466,7 +23468,7 @@ static bool setting_append_list(
          break;
       case SETTINGS_LIST_LAKKA_SERVICES:
          {
-#if defined(HAVE_LAKKA)
+#ifndef HAVE_LAKKA
             START_GROUP(list, list_info, &group_info,
                   msg_hash_to_str(MENU_ENUM_LABEL_VALUE_LAKKA_SERVICES),
                   parent_group);
